### DISCUSSION TITLE
Add a full colour gamut picker for text colour

### DISCUSSION THREAD

**Discussion by ftessier** [hakimel/reveal.js#3509] [Ideas]
The ability to select colour for spans of text inside the text blocks is very useful! It would be even better though if the text block editor colour picker included a full rgb colour picker, similar to the one for objects.

If that is not practical to implement, I would suggest that the default colour swatches for text spans match the ones for objects!

_(I understand that I can simply edit `<span>` colour inside the slide's html, but this becomes tedious!)_

![image](https://github.com/hakimel/reveal.js/assets/1444241/90edd894-72b1-4947-aa2f-ab5276a6eff6)

![image](https://github.com/hakimel/reveal.js/assets/1444241/b11b9320-22f4-4593-a1b7-7cbcf97d1f51)

**Comment by t-fritsch** [comment]
Hello !

Correct me if I'm wrong but I think the feature you're referring to is on https://slides.com/ website and not on reveal.js itself ?
slides.com is a commercial product using reveal.js, so this repo isn't the appropriate place to submit issues on this product I think.

I'll let @hakimel confirm, but I think you should use the contact form on https://slides.com/ website.

**Reply by t-fritsch** [reply]
ok I didn't look right, this isn't an issue but a discussion, I should get some sleep 😅

**Reply by ftessier** [reply]
Indeed, this is a suggestions pertaining to the slides.com editor, not reveal.js per se. I find this discussion space on github much more conducive to exchanging ideas.

**Reply by t-fritsch** [reply]
I understand your point, and saw your other discussion about using github for slides.com, thanks for that. Imho, slides.com content shouldn't live here so that open-source contributors aren't bothered with discussions out of their scope ;) anyway lets wait for clarification on  https://github.com/hakimel/reveal.js/discussions/3513
